svg.go

```diff
@@ -13,7 +14,7 @@ const svgNamespace = "http://www.w3.org/2000/svg"
 // the <svg></svg> tags, which would be invalid in this context (such as
 // <?xml...?> decls, directives, etc), writing out to a writer.  In
 // addition, loosely check that the icon is a valid SVG file.
```

Comment:
It would be nice if the comment mentioned the implications and
use of the id argument.